worker/create_batches_worker.go

```diff
@@ -252,10 +252,14 @@ func (b *CreateBatchesWorker) Process(message *workers.Msg) {
 		zap.Int("totalParts", msg.TotalParts),
 	)
 
+	b.Workers.Statsd.Incr(CreateBatchesWorkerStart, msg.Job.Labels(), 1)
+
 	err = b.Workers.MarathonDB.Model(&msg.Job).Column("job.status", "App").Where("job.id = ?", msg.Job.ID).Select()
-	checkErr(l, err)
+	b.checkErr(&msg.Job, err)
```

Comment:
it looks like you've changed the function checkErr to receive a job instead a logger, but maybe you forgot to push the worker/util.go file

Reply:
Actually, I've created a `checkErr` function in each worker which will call the `utils` one and also generate an error metric.